targets/psa_switch/psa_switch.cpp

```diff
@@ -337,6 +337,21 @@ PsaSwitch::ingress_thread() {
     parser->parse(packet.get());
     ingress_mau->apply(packet.get());
 
+    // RESUBMIT
+    if (phv->has_field("psa_ingress_output_metadata.resubmit")) {
```

Comment:
we can actually skip the `has_field` check since we know that it's guaranteed to exist by our `add_required_field`